resources/views/internlink/policy.blade.php

```diff
@@ -0,0 +1,21 @@
+@extends('layouts.panel')
+@section('panel-cols', '10')
+@section('panel-offset', '1')
+@section('panel-heading', 'InternLink')
+@section('panel-body')
+    <h2>Usage Policy</h2>
+    <h4>InternLink is meant to be a tool for current CUMSA members (Users) to seek the advice of past/current members (Contributors)
+        with regards to internship and job applications. Below are the privacy policy for the platform.</h4>
```

Comment:
* InternLink is meant to be a tool -> InternLink is a tool
* Below are the privacy policy for the platform. -> [snip, doesn't add value; also, should 'are' be 'is'?]